subsys/dm/Kconfig

```
@@ -4,17 +4,15 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-config DM_MODULE
-	bool "DM (Distance Measurement) module [EXPERIMENTAL]"
+menuconfig DM_MODULE
+	bool "nRF DM (Distance Measurement) [EXPERIMENTAL]"
```

Maybe something like that:
```suggestion
	bool "nRF DM (Distance Measurement) module [EXPERIMENTAL]"
```
Similar to this: https://github.com/nrfconnect/sdk-nrfxlib/blob/main/nrf_dm/Kconfig#L8
Also to distinguish a library from module.

---

What is the distinction between a library and a module?

---

The library is in the nrfxlib repository. Module in subsys integrates this library with NCS.

---

I can do that change, but the usage of "module" sounds a bit storage here.

In my opinion the difference between library and module as you intend it is not clear to the user.
I believe this term is used for other things elsewhere -- what do we call components under `/nrf/modules`?
Are those modules in the same way that this component is a module?

But then from a user perspective the difference is minimal anyhow, this "module" is what pulls in the library and enable its API usage in the application. From a user perspective I argue that enabling this is equivalent to adding the library to their project, so I argue that the prompt in the menu doesn't need to differentiate between "library" or "module" it could just say `nRF DM (Distance measurement)` -- you select that and you can use the API provided by the distance measurement library.

Food for thoughts, I can keep "module" in the prompt if you want, but I think this is a good chance to think about it.

---

I see, you're right, let's remove "module" in the prompt.